src/styles/card.scss

```
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 42px;
+  }
+
+  &__name {
+    margin: 0;
+    margin-bottom: 5px;
+
+    line-height: 15px;
+    font-size: 12px;
+    font-weight: 500;
+  }
```

remove unnecessary indentation
```suggestion
  &__name {
    margin: 0;
    margin-bottom: 5px;
    line-height: 15px;
    font-size: 12px;
    font-weight: 500;
  }
```